src/__tests__/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].test.tsx

```diff
@@ -330,11 +330,11 @@ describe("pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons/[le
       expect(seo).toEqual(
         expect.objectContaining({
           title:
-            "Lesson: Adverbial complex sentences | Higher | AQA | KS2 English | NEXT_PUBLIC_SEO_APP_NAME",
+            "Adverbial complex sentences Higher | AQA KS2 | Y3 English Lesson Resources | NEXT_PUBLIC_SEO_APP_NAME",
```

Comment:
I think the pipe between the tier and the exam board here should be omitted, to be consistent with the treatment where there is only a tier (line 293)
So it would read: `Adverbial complex sentences Higher AQA KS2 | Y3 English ... `

For example, this https://deploy-preview-3464--oak-web-application.netlify.thenational.academy/teachers/programmes/combined-science-secondary-ks4-higher-edexcel/units/atomic-structure-and-the-periodic-table/lessons/atomic-structure-negligible-electron-mass would read `Atomic structure (negligible electron mass) Higher Edexcel KS4 | Y10 Combined science Lesson Resources | Oak National Academy`